packages/manager/modules/billing/src/translations/Messages_en_GB.json

```diff
@@ -180,7 +180,7 @@
   "ovhAccount_create_alert": "Set up an alert",
   "ovhAccount_create_alert_resume": "You will be alerted by e-mail when your account balance falls below the alert threshold.",
   "ovhAccount_delete_alert": "Disable balance alerts on your account:",
-  "ovhAccount_create_alert_amount": "Altert threshold amount ",
+  "ovhAccount_create_alert_amount": "Alert threshold amount ",
```

Comment:
**issue:** Must be fixed via a translation request.

Reply:
Translation request was raised and has been resolved here: https://projects.dsi.ovh/browse/TRANS-39056.
So this has been added in the entry so this should be good ?

Reply:
Yes it is!

I was confused due to the commit message.
Translations used to be retrieved via a given commit message pattern which is not the case here (maybe due to a squash/rebase)

> fix(i18n): add missing translations [CDS XXXX]